listener.go

```diff
@@ -127,15 +136,25 @@ func (h *apigeeSyncHandler) processChangeList(changes *common.ChangeList) {
 		}
 	*/
 
+	// insert
 	for i := range insertedDeployments {
 		go h.bundleMan.queueDownloadRequest(&insertedDeployments[i])
 	}
 
-	// clean up old bundles
+	// update
+	for i := range updatedNewBlobs {
+		go h.bundleMan.enqueueRequest(h.bundleMan.makeDownloadRequest(updatedNewBlobs[i]))
```

Comment:
update DB table metadata_runtime_entity_metadata is done by ApigeeSync. When apidGatewayConfDeploy receives changelist, the db entries have been updated. This handler is only downloading/deleting files.